app/views/businesses/new.html.erb

```diff
@@ -0,0 +1,27 @@
+<div class="container mt-4">
+  <div class="row justify-content-center">
+    <div class="col-6">
+      <h1 class="text-center">Add a business</h1>
+      <p class="text-center">----------</p>
+      <%= simple_form_for @business do |f| %>
+         <div class="form-inputs">
+          <%= f.input :name %>
+          <%= f.input :address %>
+          <%= f.input :short_bio %>
+          <%= f.input :long_bio %>
+          <%= f.input :owner_name %>
+          <%= f.input :owner_bio%>
+          <%= f.input :telephone %>
+          <%= f.input :email%>
+          <%= f.input :opening_hours%>
+          <%= f.input :website_url%>
+          <%= f.input :photo, as: :file %>
```

Comment:
Not sure if this will break anything because we don't have :photo in the model yet. Want to leave it out for now and whoever is changing the models to add the  :photo later?